package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "axios-hooks",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

Comment:
no need to bump the version, it will be done before releasing, until then no bumps